conf/metrics.conf

```diff
@@ -833,8 +833,9 @@ metric {
 
     group "phishing" {
         symbol "PHISHING" {
-            weight = 3.0;
```

Comment:
I have some larger collections of redirectors and I **really**, no **REALLY** should write some documentation about rspamd url redirector that has been used for many years in Rambler and proved its efficiency.